apps/frontend-pwa/src/pages/course/[courseId]/element/[id].tsx

```diff
@@ -42,14 +56,18 @@ const ORDER_TYPE_LABEL = {
 interface Props {
   courseId: string
   id: string
+  description: string
+  submitText: string
 }
 
 // TODO: leaderboard and points screen after all questions have been completed?
 // TODO: different question types (FREE and RANGE)
-function LearningElement({ courseId, id }: Props) {
+function LearningElement({ courseId, id, description, submitText }: Props) {
   const router = useRouter()
   const [response, setResponse] = useState<{} | number[] | string | null>(null)
   const [currentIx, setCurrentIx] = useState(-1)
+  const [isOpen, setIsOpen] = useState(false)
+  const [open, setOpen] = useState(false)
```

Comment:
We should probably rename those states so that it is clear from the respective name, which one is used for the Modal and which one is used for the Toast.

Reply:
renamed states